buildSrc/src/main/java/Dependencies.kt

```diff
@@ -20,7 +20,7 @@ object Versions {
     const val detekt = "1.19.0"
     const val jna = "5.8.0"
 
-    const val androidx_compose = "1.1.1"
```

Comment:
We don't use alpha versions in the tree.

Reply:
I know @gabrielluong ☺️ 

This is only a check, we just want to see if the upcoming release will truly fix some inconsistencies we encounter from time to time in our UI tests.